app/src/main/java/com/ss20/se2/monopoly/view/GameboardActivity.java

```diff
@@ -158,7 +161,31 @@ public void performAcquiringDeed(Street street, Player player){
 		player.updateBalance(newBalance);
 		view_balance.setText(getString(R.string.balance,  newBalance));
 		Toast.makeText(this, "You now own " + street.getName(), Toast.LENGTH_SHORT).show();
+		showDifference(getOldBalance(), player.getBalance());
+
+	}
+	public void showDifference(int oldBalance, int newBalance){
+		int difference = oldBalance-newBalance;
+
+		if (oldBalance-newBalance == 0){
+			updateBalance.setText("");
+		}
+		else if (oldBalance<newBalance){
+			//set Color
+			updateBalance.setText("$" + difference);
```

Comment:
add `+` before $ so the User is aware that his balance has been increased. And that we have the same "Look" like in the else cond below.